### PR TITLE
chore: update dev branch version to 4.0.1-dev

### PR DESCRIPTION
**Pull request by fzipi**
- update version to next dev

### CODE REVIEW

**Review by theseion — APPROVED** [review]

**Review by RedXanadu — CHANGES_REQUESTED** [review]
There's an error. I guess there's a regex with a pattern that is looking for
`4.0.0`
and it hooked into a rule ID by mistake?

(Looks like it's rule 941010, which does indeed match that pattern.)

File: rules/REQUEST-941-APPLICATION-ATTACK-XSS.conf [hunk 1]
```diff
@@ -55,7 +55,7 @@ SecRule TX:DETECTION_PARANOIA_LEVEL "@lt 1" "id:941012,phase:2,pass,nolog,skipAf
 # support this feature at runtime.
 #
 SecRule REQUEST_FILENAME "!@validateByteRange 20, 45-47, 48-57, 65-90, 95, 97-122" \
-    "id:941010,\
+    "id:94.0.1-dev,\
```
Comment (theseion) [inline rules/REQUEST-941-APPLICATION-ATTACK-XSS.conf:58]:
```suggestion
    "id: 941010,\
```

File: rules/REQUEST-949-BLOCKING-EVALUATION.conf [hunk 2]
```diff
@@ -97,7 +97,7 @@ SecAction \
 # Summing up the blocking and detection anomaly scores in phase 2
 
 SecRule TX:BLOCKING_PARANOIA_LEVEL "@ge 1" \
-    "id:949060,\
+    "id:94.0.1-dev,\
```
Comment (theseion) [inline rules/REQUEST-949-BLOCKING-EVALUATION.conf:100]:
```suggestion
    "id: 949060,\
```

**Review by theseion — CHANGES_REQUESTED** [review]

File: rules/REQUEST-941-APPLICATION-ATTACK-XSS.conf (hunk 1 above)
Comment (RedXanadu) [reply rules/REQUEST-941-APPLICATION-ATTACK-XSS.conf]: Will the added whitespace upset the linter?

Comment (fzipi) [reply rules/REQUEST-941-APPLICATION-ATTACK-XSS.conf]: Thanks, changed.

**Review by theseion — APPROVED** [review]

**Pull request merged**